dotCMS/src/main/java/com/liferay/portlet/PortletUtil.java

```diff
@@ -0,0 +1,94 @@
+package com.liferay.portlet;
+
+import com.dotcms.repackage.org.directwebremoting.WebContext;
+import com.dotcms.repackage.org.directwebremoting.WebContextFactory;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.business.web.WebAPILocator;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.portlets.user.ajax.UserAjax;
+import com.dotmarketing.util.SecurityLogger;
+import com.liferay.portal.PortalException;
+import com.liferay.portal.SystemException;
+import com.liferay.portal.model.User;
+import javax.servlet.http.HttpServletRequest;
+
+/**
+ * @author Jonathan Gamba 11/29/17
+ */
+public class PortletUtil {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)